src/containers/CtypeCreate/CtypeCreate.tsx

```diff
@@ -1,4 +1,5 @@
-import { BlockchainUtils, CType } from '@kiltprotocol/sdk-js'
+import { CType } from '@kiltprotocol/sdk-js'
```

Comment:
`export default connect(mapStateToProps)(withRouter(CTypeCreate))` -->
There is even another one of these lines at the bottom of this file, so the other one at the top of the file should also be removed. Is eslint not complaining here?

Reply:
GitHub playing tricks on us.

Reply:
seems to be a git/github bug?